static/skin/taskbar.css

```diff
@@ -170,7 +177,7 @@ a.suggest, a.suggest:visited, a.suggest:hover, a.suggest:active {
     }
 
     .kiwix_searchform {
-        width: 36%;
+        width: 50%;
```

Comment:
Even if possibly an improvement in responsiveness, it is rather a matter of taste and has nothing to do with the issue that this PR addresses. I am not aware of the reasons behind the current value of this setting but it looks like it wasn't just random. So I am in favour of preserving it.